### DISCUSSION TITLE
etcp  打洞相关

### DISCUSSION THREAD

**Discussion by ANGJustinl** [natfrp/wiki#363] [水深火热]
- [x] 以阅读文档并确定文档无法解决问题

nat类型检测正常  我自己能确定是PortRestrictedCone
报错只有一句 detect port failed 也去frp的issue看了，但我没法确定问题，有没有人分析一下

**Comment by fakeboboliu** [comment]
您好，此问题应当是因为您到探测服务器的连接存在问题，
我们将继续优化这一功能。
